tests/rule_based_profiler/expectation_configuration_builder/test_default_expectation_configuration_builder.py

```diff
@@ -199,6 +200,67 @@ def test_default_expectation_configuration_builder_alice_null_condition(
     assert expectation_configuration.kwargs["min_value"] == 397433
 
 
+def test_default_expectation_configuration_builder_alice_null_condition_parameter_builder_validation_dependency_included(
+    alice_columnar_table_single_batch_context,
```

Comment:
Think should should go to integration tests due to multiple moving pieces and I/O with Alice fixture.

Reply:
@cdkini We can do this as a separate PR (along with other tests that are similarly inefficient).  Thanks.